glide.yaml

```diff
@@ -37,7 +37,7 @@ import:
 - package: github.com/uber/jaeger-client-go
   version: '>=1, <3'
 - package: github.com/uber/tchannel-go
-  version: ^1.4
+  version: peer-status-change-on-close
```

Comment:
Yes, note "WIP" in PR title and "TEMP" in commit message. This commit is isolated to be rebased out and replaced with a version bump on TChannel. https://github.com/uber/tchannel-go/pull/624